### PR TITLE
typo fix for pegen meta parser

### PR DESCRIPTION
**Pull request by yf-yang**
Follows pegen fix https://github.com/we-like-parsers/pegen/pull/102

### CODE REVIEW

File: Tools/peg_generator/pegen/c_generator.py [hunk 1]
```diff
@@ -167,7 +167,7 @@ def visit_NameLeaf(self, node: NameLeaf) -> FunctionCall:
                 )
             return FunctionCall(
                 assigned_variable=f"{name.lower()}_var",
-                function=f"_PyPegen_expect_token",
+                function="_PyPegen_expect_token",
```
Comment (nineteendo) [inline Tools/peg_generator/pegen/c_generator.py]: Are these f-string changes necessary? We don't make stylistic changes to untouched code.

Comment (yf-yang) [reply Tools/peg_generator/pegen/c_generator.py]: I guess it is unintentionally changed by formatter/linter somehow

Comment (nineteendo) [reply Tools/peg_generator/pegen/c_generator.py]: Then disable your linter when modifying CPython. Please revert these lines.

File: Tools/peg_generator/pegen/python_generator.py [hunk 2]
```diff
@@ -333,7 +333,7 @@ def visit_Alt(self, node: Alt, is_loop: bool, is_gather: bool) -> None:
 
                 if is_loop:
                     self.print(f"children.append({action})")
-                    self.print(f"mark = self._mark()")
+                    self.print("mark = self._mark()")
```
Comment (nineteendo) [inline Tools/peg_generator/pegen/python_generator.py]:
```suggestion
                    self.print(f"mark = self._mark()")
```

**Comment by nineteendo** [comment]
Could you avoid force pushing when not strictly necessary? Also note that suggestions can be applied directly from GitHub.

> In order to keep the commit history intact, please avoid squashing or amending history and then force-pushing to the PR. Reviewers often want to look at individual commits.

File: Tools/peg_generator/pegen/python_generator.py (hunk 2 above)
Comment (nineteendo) [reply Tools/peg_generator/pegen/python_generator.py]: You can close this.

**Comment by yf-yang** [comment]
Sure but seems doesn't matter in this PR?

**Review by lysnikolaou — APPROVED** [review]
LGTM! Thanks @yf-yang!

**Pull request merged**